src/Framework/Framework/Resources/Scripts/api/api.ts

```diff
@@ -29,7 +30,8 @@ export function invoke<T>(
     refreshTriggers: (args: any[]) => Array<KnockoutObservable<any> | string>,
     notifyTriggers: (args: any[]) => string[],
     element: HTMLElement,
```

Comment:
I'd rename this one to `cacheElement` or something, and it should be nullable. I was actually quite confused about how it works when I was going through this function last week